internal/api/api.go

```diff
@@ -120,36 +129,38 @@ func handleLogin(conn wstransport.Conn, jujuAddrs []string, jujuCert string, all
 // the provided LXD parameters. Example request/response:
 //     --> {"operation": "start"}
 //     <-- {"code": "ok", "message": "session is ready"}
-func handleStart(conn wstransport.Conn, lxd LXDParams, info *juju.Info, creds *juju.Credentials) (addr string, err error) {
+func handleStart(conn wstransport.Conn, lxd LXDParams, info *juju.Info, creds *juju.Credentials) (name, addr string, err error) {
```

Comment:
Is there a reason we're not doing `name string` in the return args as we do `addr string`? I'm still not clear on when types are or are not omitted in all these bits.

Reply:
Type applies to all variables leading up to it which aren't otherwise typed. So `a, b string, c int, d, e, f bool` is go-ish.